cmd/site-controller/README.md

```diff
@@ -0,0 +1,54 @@
+# Skupper Site Controller
+
+The site controller provides declarative methods to manage a Skupper VAN site using:
+
+* Kubernetes ConfigMaps
+* Tokens
+
+
+## Managing a Skupper Site using ConfigMaps
+
+ConfigMaps allow you to manage a Skupper site using the following parameters:
+
+`metadata:name` -  the site controller name in OpenShift
+
+`data:name` -  the VAN deployment or site name
+
+`data:cluster-local` -  (true/false) Set up skupper to only accept connections from within the local cluster.
+
+`data:console` -  (**true**/false) Enable skupper console
+
+`data:console-authentication` -  ('openshift', 'internal', 'unsecured') Autentication method
+
+`data:console-user` -  username for 'unsecured' option
+
+`data:console-password` - password for 'unsecured' option
+
+`data:edge` -  (true/false) Set up an edge skupper site
+
+`data:router-console` - (true/false) Set up a Dispatch Router console (not recommended)
+
+`data:service-controller` - 
+
+`data:service-sync` -  
+
+
+For example:
+
+```
+apiVersion: v1
+data:
+  cluster-local: "false"
+  console: "true"
+  console-authentication: internal
+  console-password: "barney"
+  console-user: "rubble"
+  edge: "false"
+  name: skupp3r
+  router-console: "true"
+  service-controller: "true"
+  service-sync: "true"
+kind: ConfigMap
+metadata:
+  name: skupper-site
```

Comment:
It would be helpful to document that the presence of this label is what causes the site-controller to take action on this config map. Without the label, the site-controller will filter out evaluation of the cm.

Reply:
@ajssmith added note